docs/source/run.rst

```diff
@@ -64,6 +64,10 @@ Commonly Used Arguments
   adjust the name of Gunicorn process as they appear in the process system
   table (which affects tools like ``ps`` and ``top``).
 
+.. Gevent note::
```

Comment:
No. This problem is not just with Gevent worker. Generally using
monkey.patch_all leads to problems. If my view is correct Benoit's
suggestion to add in run.rst is better. Give me feedback again.

On Sat, Oct 3, 2015 at 7:12 PM, Benoit Chesneau <notifications@github.com>
wrote:

> In docs/source/run.rst
> <https://github.com/benoitc/gunicorn/pull/1121#discussion_r41088272>:
>
> > @@ -64,6 +64,10 @@ Commonly Used Arguments
> >    adjust the name of Gunicorn process as they appear in the process system
> >    table (which affects tools like ``ps`` and ``top``).
> >
> > +.. Gevent note::
>
> that's probably better indeed
> … <#1502e5e18f8ac46c_>
> On Sat, 3 Oct 2015 at 17:39, Berker Peksag <notifications@github.com>
> wrote: In docs/source/run.rst <
> https://github.com/benoitc/gunicorn/pull/1121#discussion_r41088242>: > @@
> -64,6 +64,10 @@ Commonly Used Arguments > adjust the name of Gunicorn
> process as they appear in the process system > table (which affects tools
> like ``ps`` and ``top``). > > +.. Gevent note:: .. note:: Foo bar [...] —
> Reply to this email directly or view it on GitHub <
> https://github.com/benoitc/gunicorn/pull/1121/files#r41088242>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benoitc/gunicorn/pull/1121/files#r41088272>.
>